other-examples/collector/hcp-consul/collector.yaml

```diff
@@ -0,0 +1,34 @@
+receivers:
+  prometheus:
+    config:
+      global:
+        scrape_interval: "60s"
+      scrape_configs:
+        - job_name: "hcp-consul-cluster"
+          scheme: "https"
+          dns_sd_configs:
+            - names:
+                - "$HCP_ACCESS_URL"
+              type: "A"
+              port: 443
+          authorization:
+            credentials: "$HCP_ACCESS_TOKEN"
+          metrics_path: "/v1/agent/metrics"
+          tls_config:
+            insecure_skip_verify: true
```

Comment:
Why do we want to skip tls verification?

Reply:
This is the recommended setting in HCP docs, so it's what I used. I'm not sure if it's required or not, but even their docs for scraping from self-managed API's includes this setting.

https://developer.hashicorp.com/hcp/docs/consul/monitor/metrics#prometheus

Reply:
Can you try running without this option? If required, can you include a comment pointing to these docs for this?